.travis.yml

```diff
@@ -4,5 +4,10 @@ python:
   - "2.7"
   - "3.3"
   - "3.4"
-install: "pip install -r requirements.txt"
-script: py.test 
+install:
+  - pip install -r requirements.txt
+  - pip install codecov pytest-cov
```

Comment:
I'm not sure how I missed that...Sorry. For sure move the into the requirements file.